data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/node/GetHealth.java

```diff
@@ -78,21 +90,26 @@ public GetHealth(final DataProvider provider) {
   @Override
   public void handle(@NotNull final Context ctx) throws Exception {
     ctx.header(Header.CACHE_CONTROL, CACHE_NONE);
+    adapt(ctx);
+  }
+
+  @Override
+  public void handleRequest(RestApiRequest request) throws JsonProcessingException {
     if (!chainDataProvider.isStoreAvailable()) {
-      ctx.status(SC_SERVICE_UNAVAILABLE);
+      request.respondError(SC_SERVICE_UNAVAILABLE, "Node not initialized or having issues");
     } else if (syncProvider.isSyncing()) {
       int syncingStatus = SC_PARTIAL_CONTENT;
       try {
-        final Map<String, List<String>> parameters = ctx.queryParamMap();
+        final Map<String, List<String>> parameters = request.getQueryParamMap();
         if (!parameters.isEmpty()) {
           syncingStatus = getParameterValueAsInt(parameters, SYNCING_STATUS);
         }
       } catch (final IllegalArgumentException ex) {
         LOG.trace("Illegal parameter in GetHealth", ex);
       }
-      ctx.status(syncingStatus);
+      request.respondError(syncingStatus, "Node is syncing but can serve incomplete data");
```

Comment:
i think to be consistent, we're probably passing back an empty string in all these responds... currently this is likely to return a body and this endpoint I think only previously set status...